.github/workflows/release.yml

```
@@ -0,0 +1,108 @@
+
+      - name: Move RPM
+        run: mv *.rpm dist/ 2>/dev/null || true
+
+      # Build Arch package
+      - name: Build Arch Package
+        uses: FFY00/build-arch-package@v1
+        with:
+          PKGBUILD: dkms/PKGBUILD
+
+      - name: Move Arch Package
+        run: mv *.pkg.tar.zst dist/ 2>/dev/null || true
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2035:info:1:4: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)